public/pages/MonitorDetails/containers/MonitorDetails.js

```diff
@@ -492,12 +492,15 @@ export default class MonitorDetails extends Component {
 
     if (useUpdatedUx) {
       monitorActions.unshift(
-        <EuiSmallButton
-          iconType={'trash'}
-          onClick={this.onDeleteClick}
-          color="danger"
-          aria-label="Delete"
-        ></EuiSmallButton>
+        <EuiToolTip content="Delete">
+          <EuiSmallButton
```

Comment:
did we figure out the spacing for this one?

Reply:
Yes can see what the button looks like under the view monitor page image